controllers/tab/catalogEntry/form/CatalogEntryCatalogMetadataForm.inc.php

```diff
@@ -67,10 +67,14 @@ function __construct($monographId, $userId, $stageId = null, $formParams = null)
 	 */
 	function fetch($request, $template = null, $display = false) {
 		$templateMgr = TemplateManager::getManager($request);
-		$templateMgr->assign('submissionId', $this->getMonograph()->getId());
+		$monograph = $this->getMonograph();
+		$templateMgr->assign('submissionId', $monograph->getId());
 		$templateMgr->assign('stageId', $this->getStageId());
 		$templateMgr->assign('formParams', $this->getFormParams());
-		$templateMgr->assign('datePublished', $this->getMonograph()->getDatePublished());
+		$templateMgr->assign('datePublished', $monograph->getDatePublished());
+		$chapterPublicationDatesEnabled = (int)$monograph->getEnableChapterPublicationDates() == 1;
+		$templateMgr->assign('enableChapterPublicationDates', $chapterPublicationDatesEnabled);
```

Comment:
Same here, let's keep it all at `enableChapterPublicationDates`.

Reply:
https://github.com/pkp/omp/pull/687/commits/43f2e36adad93c38b7b6bd6c32f65e56e0c27985